modules/consensus/database.go

```diff
@@ -21,7 +23,7 @@ var (
 	errNonEmptyBucket = errors.New("cannot remove a map with objects still in it")
 
 	dbMetadata = persist.Metadata{
-		Version: "0.4.3",
+		Version: "0.5.0",
```

Comment:
Yeah, the changelog is a breaking change.

If you feel strongly about it, I can implement a separate upgrade pathway that'll add the changelog without needing to redownload everything, but that's going to be a nontrivial change, and code only used once.

Right now, IBD takes less than 4 hours even on an HDD, and it's like 10 minutes on an SSD.